index.html

```diff
@@ -10,14 +10,15 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div id="header">
-      The HITMAN Hat Game
-    </div>
+    <div id="header"><img src=hitman_logo.jpg width=500></div>
     <div id="content">
+        Pick your stage, targets, weapon, and disguise.</p>
+        <p>Feeling dangerous? Pick a wildcard.</p>
+      <p></p>
```

Comment:
Is this paragraph just here for spacing? If so, consider doing the spacing in CSS.